lib/elasticsearch.ts

```diff
@@ -176,16 +176,30 @@ function filter_managed_aliases(aliases) {
 }
 
 function update_alias_state(current_indices, alias) {
+  const prefix = config.indices.prefix;
   return new Promise((resolve, reject) => {
     const acceptable_indices = [];
     let today = moment();
     for (let i = 0; i < config.aliases.mappings[alias]; i++) {
-      acceptable_indices.push(`${config.indices.prefix}-${today.format(constants.date_format)}`);
+      const suffix = today.format(constants.date_format);
+
+      // always add today
+      acceptable_indices.push(
+        `${config.indices.prefix}-kvm-${today.format(constants.date_format)}`,
+        `${config.indices.prefix}-${today.format(constants.date_format)}`,
+      );
+
+      acceptable_indices.push(
+        ...current_indices.filter(index => index.startsWith(prefix) && index.endsWith(suffix))
+      );
```

Comment:
add comment about what this does? It looks like this is new logic, and I'm not entirely sure what the purpose is

Reply:
Yeah, not sure I understand this line either. Seems like this would add values that are already in `acceptable_indices`?

Reply:
You're right, I tried this before realizing that current indices does not always contain todays index so we need to manually add them.

This would prevent against adding new `logs-*-<date>` into the index and that not showing up for multiple days.

Reply:
added comment about keeping manually added indexes to alias in https://github.com/Clever/elasticsearch-toolbox/pull/33/commits/78f89f86637a95f29079c91c763c4f241277dc25